src/test/java/apicurito/tests/steps/VerificationSteps/CollectorHelper.java

```diff
@@ -0,0 +1,10 @@
+package apicurito.tests.steps.VerificationSteps;
+
+import org.junit.Rule;
+
+import org.assertj.core.api.SoftAssertions;
+
+public class CollectorHelper {
+    @Rule
+    protected static SoftAssertions collector = new SoftAssertions();
```

Comment:
Why do you need static collector? Also why is it protected?

Reply:
Because non static do not work as I need. 
protected is because of inheritance

Reply:
Ok more strict questions: why do we need collector at all? Why do you introduce such inheritance for a static variable?

Reply:
Because of usage SoftAssert, not classic Assert. I want one collector which is used (it collects all asserts) in all verification classes.

Reply:
I can see why you want that, I do not like it as the tests should be independent, but in some cases you do not want to fail the test just because wrong tooltip. Anyway - the variable is static, you do not need inheritance at all - it is accessible from everywhere, isn't it? That is what hurts my eyes. Please consider where the collector is really necessary and get rid of that inheritance.

Reply:
updated: now without inheritance